bepasty_cli/cli.py

```diff
@@ -55,10 +59,7 @@ def main(token, fileobj, fname, url, ftype):
         stdin = False
 
     else:
-        #  tried to write stdin in tempfile and run guess_file, but guess_file
-        #  only seem to look on the file extension ...
-        # tmpfile = NamedTemporaryFile(delete=False)
-        fileobj = click.get_binary_stream('stdin')
+        fileobj = BytesIO(click.get_binary_stream('stdin').read())
```

Comment:
fileobj is a misleading name here, as it is the file's content (here: everything it has read from stdin).